app/MMB/src/list/FlightList.js

```diff
@@ -31,6 +33,7 @@ export class FlightList extends React.Component<Props> {
             type={type}
             imageUrl={imageUrl}
             booking={idx(item, _ => _.node.oneWay)}
+            isPastFlight={isPastFlight}
```

Comment:
This is another indicator of odd design - these conditional props affecting internal logic. Just keep it simple and use one component in multiple places... :)